MCprep_addon/mcprep_ui.py

```diff
@@ -2093,13 +2068,9 @@ def register():
 		default=addon_prefs.custom_texturepack_path)
 
 	env.verbose = addon_prefs.verbose
-	if hasattr(bpy.types, "INFO_MT_add"):  # 2.7
-		bpy.types.INFO_MT_add.append(draw_mcprepadd)
-	elif hasattr(bpy.types, "VIEW3D_MT_add"):  # 2.8
+	if hasattr(bpy.types, "VIEW3D_MT_add"):  # 2.8
```

Comment:
Here and below, might be god to have a printout in an else clause - that way there still isn't a registration error, but we'd be aware of it when looking at startup logs.

Also in unregister, a few remaining blocks which can be simplified to remove the 2.7 logic branches. Same deal to just have an else clause that prints a warning.